Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM enmasseproject/java-base:11-0
+
```

Comment:
This should be using quay.io/enmasse/java-base:11-1

Reply:
Thanks for the catch, I copied from a stale branch in my workspace in error.